py/legacypipe/runbrick.py

```diff
@@ -1214,8 +1220,8 @@ def stage_srcs(targetrd=None, pixscale=None, targetwcs=None,
         sat.ref_cat = np.array(['  '] * len(sat))
         del satyx
         
-        avoid_x = np.append(avoid_x, sat.ibx)
-        avoid_y = np.append(avoid_y, sat.iby)
+        avoid_x.append(sat.ibx)
```

Comment:
wait, what type is avoid_x?  if it's a list, shouldn't it be .extend() rather than .append()?

Reply:
I was replacing 'numpy.append', which stupidly changes the datatype, but, yes, I meant to use `.extend`.  Change was committed.